examples/all-clusters-app/all-clusters-common/include/EnergyEvseDelegateImpl.h

```diff
@@ -70,7 +71,7 @@ class EnergyEvseDelegate : public EnergyEvse::Delegate
 
     // -----------------------------------------------------------------
     // Internal API to allow an EVSE to change its internal state etc
-    // TODO Status HwRegisterEvseHardwareCallback(Callback);
+    Status HwRegisterEvseCallbackHandler(EVSECallbackFunc handler, intptr_t arg);
```

Comment:
This needs to document the "you can only register one handler" limitation.

Reply:
This is sort of documented in the .cpp but added something in the .h too.